python/langsmith/evaluation/llm_evaluator.py

```diff
@@ -52,23 +87,24 @@ def _create_score_json_schema(
     else:
         raise ValueError("Invalid score type. Must be 'categorical' or 'continuous'")
 
-    if score_config.include_explanation:
-        properties["explanation"] = {
-            "type": "string",
-            "description": (
-                "The explanation for the score."
-                if score_config.explanation_description is None
-                else score_config.explanation_description
-            ),
-        }
-
     return {
         "title": score_config.key,
         "description": score_config.description,
         "type": "object",
         "properties": properties,
         "required": (
-            ["score", "explanation"] if score_config.include_explanation else ["score"]
+            [
+                (
+                    "value"
+                    if isinstance(score_config, CategoricalScoreConfig)
```

Comment:
nit: easier to read if we just separate this out above (since nested python ternary expressions are a bit daunting)